libs/jsonforms-components/src/lib/Controls/FormStepper/FormStepperControl.tsx

```diff
@@ -157,6 +226,31 @@ export const FormStepper = ({
       setShowNextBtn(true);
     }
   }
+  function getAllRequiredFields() {
+    const requiredFields: string[] = [];
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    function findRequired(fields: any) {
```

Comment:
Is it really _any_?  Maybe in this case it is, but I see you are using _any_ a lot.  Please try to determine types where possible. Types are your friend.